src/controllers/action-queue/push.js

```diff
@@ -89,11 +89,11 @@ export async function prepareNewPushEvents(
 }
 
 export async function checkPushEvents(account: EdgeAccount, eventIds: string[]): Promise<boolean> {
-  const { rootLoginId: loginId } = account
+  const { rootLoginId } = account
   const requestBody: PushRequestBody = {
     apiKey: AIRBITZ_API_KEY,
     deviceId,
-    loginId: asBase64(loginId)
+    loginId: asBase58(rootLoginId)
```

Comment:
Do not use a cleaner here, since `rootLoginId` has a known type. Instead, use `base58.parse` directly.

Reply:
That's right. Here I go again mis-using cleaners. I realized my desire for a kind of function useful for runtime type-checking vs _runtime type casting_ (which I believe is a useful description as to what cleaners do).